documentation/release_notes.markdown

```diff
@@ -10,6 +10,18 @@ canonical: "/puppetserver/latest/release_notes.html"
 [puppetserver.conf]: ./config_file_puppetserver.markdown
 [product.conf]: ./config_file_product.markdown
 
+## Puppet Server 6.11.1
+
+Released 7 May 2020
+
+### Resolved Issue
```

Comment:
Think we should move both notes under "known issue"? A rollback doesn't feel like a resolution, and the ticket isn't closed.

Reply:
works for me